web-api/src/transaction/transaction.service.ts

```diff
@@ -184,10 +184,27 @@ export class TransactionService {
 		entity.txRaw = txRaw;
 		entity.isCentralized = isFallback;
 
-		const newBridgeTransaction =
+		var newBridgeTransaction =
 			this.bridgeTransactionRepository.create(entity);
-		await this.bridgeTransactionRepository.save(newBridgeTransaction);
 
-		return mapBridgeTransactionToResponse(newBridgeTransaction);
+		try {
+			await this.bridgeTransactionRepository.save(newBridgeTransaction);
```

Comment:
do:
await this.bridgeTransactionRepository.save(newBridgeTransaction);
return mapBridgeTransactionToResponse(newBridgeTransaction);